### ISSUE TITLE
Error when sending the feedback form

### ISSUE THREAD

**Issue by hohse** [openqda/openqda#44]
I tried the feedback form on the staging server. I gave it the name "Test" and the text content "This is a test." and activated the check box for "Contact me", and tried to send it via the "Send" button. Then I got this error message:

<img width="275" alt="Screenshot 2024-06-11 at 11 33 36" src="https://github.com/openqda/openqda/assets/9248259/5e7445f2-7d95-4b61-bc85-f0a245e0a4b3">

I ran it on Chrome Version 125.0.6422.142 (Official Build) (arm64) under MacOS Sonoma 14.5

**Comment by jankapunkt** [comment]
This is weird. The route is explicitly post:

```
Route::post('/user/feedback', [UserNavigationController::class, 'feedback']);
```

and the request is also post

```
const response = await axios.post('/user/feedback/', payload);
```

but the error says `The GET method is not supported for route user/feedback`

**Comment by alessandrobelli** [comment]
> This is weird. The route is explicitly post:
> 
> ```
> Route::post('/user/feedback', [UserNavigationController::class, 'feedback']);
> ```
> 
> and the request is also post
> 
> ```
> const response = await axios.post('/user/feedback/', payload);
> ```
> 
> but the error says `The GET method is not supported for route user/feedback`

Axios is the problem, I have no idea why but if the url ends with / it makes problems.
I solved this with the commit 577ad669c2cf98a00c924b6bcbef4cc4e63e3dc3
